onnx/defs/math/defs.cc

```diff
@@ -644,6 +644,25 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyGelu)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* SiLU_ver21_doc = R"DOC(
+Sigmoid Linear Unit (SiLU), or known as the Swish function, takes one input data (Tensor<T>) 
```

Comment:
Since we have `HardSwish`, would it be more uniform to call this function-op `Swish` ? Just a minor matter, just wondering.

Reply:
good idea! I will update this to `swish` and add the beta parameter.